core/domain/user_jobs_one_off.py

```diff
@@ -362,9 +362,33 @@ def map(user_model):
         user_model.put()
 
 
-class CleanupActivityIdsFromUserSubscriptionsModelOneOffJob(
+class CopyActivityIdsToExplorationIdsFromUserSubscriptionsModelOneOffJob(
```

Comment:
```suggestion
class FillExplorationIdsInUserSubscriptionsModelOneOffJob(
```

Reply:
You will also need to add tests for this job

Reply:
Done.